CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Fix: alarm for notification queue overpassing a given threshold (#4113)
```

Comment:
```suggestion
- Add: alarm for notification queue overpassing a given threshold (new CLI `-notifAlarmThreshold` / env var `ORION_NOTIF_ALARM_THRESHOLD`) (#4113)
```

("Add" better than "Fix" given it is a new functionality)

Reply:
Fixed